boost-maven/boost-maven-plugin/src/it/test-jdbc/src/test/java/it/EndpointIT.java

```diff
@@ -22,7 +22,8 @@
 
     @BeforeClass
     public static void init() {
-        URL = "http://localhost:9080/";
+        String port = System.getProperty("boost.http.port");
```

Comment:
So this is a bit odd...  First, I agree that since we set a test.port we should use that in the test code. Second, this only works if boost.http.port is explicitly passed in on the command line as a -D.  That is the only way both the test code and boost plugin can use it.  Using the default (i.e. dont set anything) wont allow the test code to see it. 

In short... because we have this workaround of needing to set the port explicitly, you can functionally access the boost.http.port property from the test code.